### PR TITLE
True false Fixed

### PR DESCRIPTION
**Pull request by mehtadarshan1**
Fixes issue #145

### CODE REVIEW

**Review by ahmedqarmout2 — APPROVED** [review]
looks good

**Review by Mohamed-A-Mohamed — APPROVED** [review]
Nice work

File: views/tf-answer.pug [hunk 1]
```diff
@@ -1,31 +1,37 @@
...
-				.input-field.col.s1
-					if question.answer == option
-						input(type='radio', name='radbutton', id=i, value=i, required, checked)
-					else
-						input(type='radio', name='radbutton', id=i, value=i, required)
-					label(for=i)
-				.input-field.col.s11
-					input.form-control(type='text', name='mcans'+i, placeholder='Enter New Answer Here', required, style='float:left;' value=option)
+		.row
+			if question.answer == 'true'
```
Comment (esteeroramy) [inline views/tf-answer.pug]: `==` -> `===`

**Review by esteeroramy — APPROVED** [review]
just one minor comment, please address before merging

**Pull request merged**